operator/controllers/actionsrunner/util/converters.go

```diff
@@ -360,22 +360,41 @@ func ToJob(actionsRunner *inlocov1alpha1.ActionsRunner, actionsRunnerJob *inloco
 	return &job, nil
 }
 
-func filterResourceRequirements(source corev1.ResourceRequirements, resourceNames ...corev1.ResourceName) corev1.ResourceRequirements {
-	resourceRequirements := corev1.ResourceRequirements{
-		Limits:   corev1.ResourceList{},
-		Requests: corev1.ResourceList{},
+func withRuntimeAffinity(affinity *corev1.Affinity) *corev1.Affinity {
+	if affinity == nil {
```

Comment:
This long list of `if`s hurts my eyes x.x Unfortunately this seems like the best we can do